backend/app/core/search.py

```diff
@@ -456,6 +460,13 @@ def order_by(self, field: SortField, order: SortOrder):
         else:
             raise RuntimeError("Unknown sort ordering field: {field}")
 
+    def with_required_fields(self, required_fields: List[str] = ["document_name"]):
```

Comment:
do we want to get rid of the default value of `required_fields` here as it's hardcoded above? (i know it was in my example but it was lazy coding because it followed a bunch of other kwargs with defaults 😅)

Reply:
ugh, I meant to change that! using mutables as kwargs is really bad practice & I forgot to remove it! thanks!